### ISSUE TITLE
transformers in ktrain?

### ISSUE THREAD

**Issue by randomgambit** [amaiya/ktrain#382]
Hi @amaiya,

sorry for posting so many questions, but your package is great and I want to make sure I understand what is going on.

I am interested in text classification, so naturally I followed the tutorial here https://github.com/amaiya/ktrain/blob/master/tutorials/tutorial-A3-hugging_face_transformers.ipynb

However, where can I find more details about what happens under the hood? Right now the model performs well but by looking at the official transformers documentations there are many hyperparameters/options to choose from. What is `ktrain` doing exactly? Do we have any flexibility in terms of parameters/options when training the model (beyond choosing the transformer model)?

Thanks!

**Comment by randomgambit** [comment]
specifically, I am using the model 'distilbert-base-uncased' if that helps

**Comment by randomgambit** [comment]
for instance, would it be fair to say that running `ktrain` with `distilbert-base-uncased` would give the same results as using this tutorial? https://medium.com/geekculture/hugging-face-distilbert-tensorflow-for-custom-text-classification-1ad4a49e26a7

**Comment by amaiya** [comment]
Hello:  Answers to these kinds of questions are usually best found by examining source code and running experiments.  The tutorial you posted does not use a schedule for the learning rate and weight decay, so it may be similar to invoking `fit` instead of `fit_onecycle` in ktrain.

**Comment by randomgambit** [comment]
thank you so much, @amaiya. I appreciate your help.

**Comment by randomgambit** [comment]
i am actually quite surprised. I ran the same model using the "fine tuning" approach and I got a catastrophic accuracy score (about 40%) compared to the 85%+ I was getting with `ktrain`. Could the learning rate be so important?
